pkgs/applications/networking/irc/catgirl/default.nix

```
@@ -0,0 +1,21 @@
+{lib, stdenv, fetchurl, ncurses, libressl, man, pkg-config, ctags}:
+
+stdenv.mkDerivation rec {
+  name = "catgirl-1.6";
+
+  src = fetchurl {
+    url = "https://git.causal.agency/catgirl/snapshot/${name}.tar.gz";
+    sha256 = "0shg02zidqqmvywqqsaazlgg9rd5lhhrvjx6n0lzmdfaawxywciv";
+  };
+
+  nativeBuildInputs = [ pkg-config ncurses libressl man ctags ];
+
+  meta = with lib; {
+    homepage = "https://git.causal.agency/catgirl/about/";
+    license = lib.licenses.gpl3;
```

See line 13 (`with lib`)

```suggestion
    license = licenses.gpl3Plus;
```

The header in the source file says:

```text
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 ```